t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -1131,7 +1131,7 @@ test_expect_success 'sparse-index is expanded and converted back' '
 	GIT_TRACE2_EVENT="$(pwd)/trace2.txt" GIT_TRACE2_EVENT_NESTING=10 \
 		git -C sparse-index -c core.fsmonitor="" mv a b &&
 	test_region index convert_to_sparse trace2.txt &&
-	test_region index ensure_full_index trace2.txt
+	test_region index full_index trace2.txt
```

Comment:
I don't think this is a meaningful fix. Let's explore what's going on by pairing later today.